prow/cmd/deck/static/docs.html

```diff
@@ -0,0 +1,304 @@
+                    <p>Set the payload URL to <strong>http://&lt;IP-FROM-INGRESS&gt;/hook</strong>, the content type to
+                    <strong>application/json</strong>, the secret to your HMAC secret, and ask it to send everything.After you’ve created your webhook, GitHub will indicate that it successfully sent an event by putting a green checkmark under “Recent Deliveries.”</p>
+                    
+
+                    <h1><a id="Next_steps"></a>Next steps</h1>
+                    <h2><a id="Enable_some_plugins_by_modifying_pluginsyaml">
+                    </a>Enable some plugins by modifying <strong>plugins.yaml</strong></h2>
+                    <p>Create a file called <strong>plugins.yaml</strong> and add the following to it:</p>
+                    <pre class="line-numbers"><code class="language-yaml">plugins:
+                                  YOUR_ORG/YOUR_REPO:
+                                  - size</code></pre>
+                    <p>Replace <code class="language-yaml">YOUR_ORG/YOUR_REPO:</code> with the appropriate values. If you want, you can instead just say <code class="language-yaml">YOUR_ORG:</code> and the plugin will run for every repo in the org.</p>
+                    <p>Run the following to test the file, replacing the path as necessary:</p>
+                    
+                    <pre><code class="language-command-line">bazel run //prow/cmd/config -- --plugin-config=path/to/plugins.yaml</code></pre>
+
+                    <p>There should be no errors. You can run this as a part of your presubmit testing so that any errors are caught before you try to update.</p>
+                    <p>Now run the following to update the configmap, replacing the path as necessary:</p>
+                    <pre><code class="language-command-line">kubectl create configmap plugins -from-file=plugins=path/to/plugins.yaml --dry-run -o yaml | kubectl replace configmap plugins -f -</code></pre>
+                    
+                    <p>Now when you open a PR, it will automatically be labelled with a <strong>size/*</strong> label. When you make a change to the plugin config and push it with <strong>make update-plugins</strong>, you do not need to redeploy any of your cluster components. They will pick up the change within a few minutes.</p>
+                    
+                    <h2><a id="Add_more_jobs_by_modifying_configyaml_122">
+                    </a>Add more jobs by modifying <strong>config.yaml</strong></h2>
+                    <p>Create a file called <strong>config.yaml</strong>, and add the following to it:</p>
+
+                    <pre class="line-numbers"><code class="language-yaml">periodics:
+                                - interval: 10m
+                                  agent: kubernetes
+                                  name: echo-test
+                                  spec:
+                                  containers:
+                                   &nbsp - image: alpine
```

Comment:
nit: entities should be like `&nbsp;`

Reply:
there is some issue with the yaml formatting. I am working on it.